### PR TITLE
Improvements to Dreams image generation prompts

### PR DESCRIPTION
**Pull request by Frankie-Boy**
# Short Description
- Cretes algorithm to postfix keywords and characteristics on to the generated `title`

# What changes in the code?
- Adds newly defined list of characteristics, examples include; 
- `Genre e.g. Western`, `Century or Era e.g. 1970s`, `contentType e.g. Painting`

# Did you test?
- Yes tested locally

# TODO
- Improve UI for characteristics added

### CODE REVIEW

File: src/engineering-meeting/meeting-randomiser.ts [hunk 1]
```diff
@@ -107,11 +122,78 @@ class NameRandomiser {
...
+         return this.random(this.century)
+      }
+   }
+
+   random(array: string[]): string {
+      return array[Math.floor(Math.random() * array.length)]
+   }
+
+   coinFlip() : boolean {
+      const coin = Math.round(Math.random()) + 1
```
Comment (juankysoriano) [inline src/engineering-meeting/meeting-randomiser.ts]: `return Math.random() > 0.5`

File: src/engineering-meeting/meeting-randomiser.ts [hunk 2]
```diff
@@ -107,11 +122,78 @@ class NameRandomiser {
...
+         return this.random(this.century)
+      }
+   }
+
+   random(array: string[]): string {
+      return array[Math.floor(Math.random() * array.length)]
+   }
+
+   coinFlip() : boolean {
+      const coin = Math.round(Math.random())
```
Comment (juankysoriano) [inline src/engineering-meeting/meeting-randomiser.ts]: you can simplify with `return Math.random() > 0.5`

Comment (Frankie-Boy) [reply src/engineering-meeting/meeting-randomiser.ts]: [ed7afc8](https://github.com/novoda/engineering-meeting/pull/24/commits/ed7afc88dad148e4d71de9b670e3b2977b162abf)

File: src/engineering-meeting/meeting-randomiser.ts [hunk 3]
```diff
@@ -107,11 +122,78 @@ class NameRandomiser {
...
+
+   generateMediaType(nameTitle: string): string {
+      const genre = this.random(this.genre)
+      const type = this.random(this.contentType)
+      const timePeriod = this.generateTimePeriod()
+      let flavour = ""
+      switch(type) {
+         case "Poster": {
+            flavour = this.generatePoster()
+            break;
```
Comment (Frankie-Boy) [inline src/engineering-meeting/meeting-randomiser.ts:141]: Ideally I'd assign these from the switch and avoid having to `break;`

**Pull request merged**